organization/material/organizational_remarks_week7_slides.md

```diff
@@ -0,0 +1,64 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+  data-background-color: green
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+  .reveal code {
+    font-family: 'Ubuntu Mono';
+    color: orange;
+  }
+  .reveal section img {
+    background:none;
+    border:none;
+    box-shadow:none;
+  }
+</style>
+
+# Organizational Things
```

Comment:
Should we start numbering these headlines or give them a date? 

Something like
```markdown
# Organizational Things 2021-12-02
```
or

```markdown
# Organizational Things (3)
```